src/modules/tracker/tracker-supply.sync.ts

```diff
@@ -54,4 +67,16 @@ export class TrackerSupplySync {
 
     return trackerSupply;
   }
+
+  private async getTokenNames() {
+    const result: TokenPrice[] = await this.tokenPriceService.findAll();
+
+    const tokenNames: string[] = [];
+
+    result.forEach((token: TokenPrice) => {
+      tokenNames.push(token.token);
+    });
+
+    return tokenNames;
+  }
 }
```

Comment:
The function could be written in a more readable way: 

```suggestion
  private async getTokenNames(): Promise<string[]> {
    const tokenPrices = await this.tokenPriceService.findAll();
    
    return tokenPrices.map(tokenPrice => tokenPrice.token);
  }
}
```